### DISCUSSION TITLE
Autocomplete suggestions on all page loads?

### DISCUSSION THREAD

**Discussion by ElDani82** [searxng/searxng#1269] [Q&A] [answered]
Using autocomplete, I'm faced with this on every new page load:

![image](https://user-images.githubusercontent.com/5216053/171349849-d05b791d-c376-453b-94b4-6881e408e4c2.png)

To explain: even though I already entered the search term and am only accessing new results (whether the next page of results or a different category), the search suggestions always pop up and do so in an inconvenient location. To access a different category, I always have to click away the suggestions first.

I didn't want to open a bug report/feature request about it yet, because I'm not sure if this is its intended behavior, or if there is some other setting I'm missing.

**Comment by mrpaulblack** [comment, marked as answer]
Hi @ElDani82 ,
there was a bug report a few days ago -> https://github.com/searxng/searxng/issues/1270
which got fixed in https://github.com/searxng/searxng/pull/1272. With this there should be no more autofocus on search results on desktop. Is your issue fixed with the latest SearXNG master? you could try it out here: https://searx.tiekoetter.com/ (just an example of an up to date instance).

**Reply by ElDani82** [reply]
Indeed, the autofocus now behaves differently and as expected. I had already noticed it in my daily use before I read your reply.

Thanks for checking back with me, @mrpaulblack .
